### PR TITLE
Feature/note 001

### PR DESCRIPTION
**Pull request by awgrass**

### CODE REVIEW

File: app/src/androidTest/java/at/tugraz/ist/swe/note/MainActivityTest.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(AndroidJUnit4.class)
+public class MainActivityTest {
+
+    @Test
+    public void checkCreateNoteButtonVisibility() {
+        //onView(withId(R.id.createNoteButton)).check(matches(isDisplayed()));
```
Comment (haubitzer) [inline app/src/androidTest/java/at/tugraz/ist/swe/note/MainActivityTest.java]: fix that ;)

File: app/src/main/java/at/tugraz/ist/swe/note/MainActivity.java [hunk 2]
```diff
@@ -1,13 +1,49 @@
...
+            }
+        });
+
+        //noteTitles should contain title of notes. Need to be dynamically loaded.
+        //This could for example be just <Note i> for i in [0 ... length if notes list]
+        //or maybe the first few words of the corresponding note.
+        String[] noteTitles = new String[]{"Note 1", "Note 2", "Note 3", "Note 4", "Note 5",
+                "Note 6", "Note 7", "Note 8", "Note 9", "Note 10", "Note 11", "Note 12"};
+
+        ArrayAdapter<String> noteListViewAdapter = new ArrayAdapter<>(this, R.layout.note_list_item, noteTitles);
```
Comment (haubitzer) [inline app/src/main/java/at/tugraz/ist/swe/note/MainActivity.java:42]: do you need to save it, because you use it only once

Comment (awgrass) [reply app/src/main/java/at/tugraz/ist/swe/note/MainActivity.java:42]: yes, I save it for aestethic reasons.

**Review by agape94 — APPROVED** [review]
Looks good to me.

**Pull request merged**